00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/update_hub GitHub Action.md

```diff
@@ -0,0 +1,24 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# update_hub GitHub Action
+
+Links to Action:
+
+- Source of the workflow: [`update_hub.yml`](https://github.com/obsidian-community/obsidian-hub/blob/main/.github/workflows/update_hub.yml)
```

Comment:
I added a link to the source of the workflow, as not everyone will know how to obtain that from looking at the logs... (I only learned that link recently...)